ui-components/src/page_components/listing/listing_sidebar/ListingUpdated.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import { t } from "../../../helpers/translator"
+import { Listing } from "@bloom-housing/backend-core/types"
+import moment from "moment"
+
+interface ListingUpdatedProps {
+  listing: Listing
+}
+
+const ListingUpdated = (props: ListingUpdatedProps) => {
+  const listing = props.listing
+  return (
+    <section className="aside-block">
+      <p className="text-tiny text-gray-800">
+        {t("listings.listingUpdated")}
```

Comment:
Would prefer to see template literals here for readability (\`${t("listings.listingUpdated")}: ${moment...}\`)

Reply:
Thanks for the suggestion.  Updated here in latest commit.